lib/src/DateRangePicker/DateRangePickerInput.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import Typography from '@material-ui/core/Typography';
+import KeyboardDateInput from '../_shared/KeyboardDateInput';
+import { RangeInput, DateRange } from './RangeTypes';
+import { useUtils } from '../_shared/hooks/useUtils';
+import { makeStyles } from '@material-ui/core/styles';
+import { MaterialUiPickersDate } from '../typings/date';
+import { DateInputProps } from '../_shared/PureDateInput';
+import { CurrentlySelectingRangeEndProps } from './RangeTypes';
+import { createDelegatedEventHandler } from '../_helpers/utils';
+
+export const useStyles = makeStyles(
+  theme => ({
+    rangeInputsContainer: {
+      display: 'flex',
+      alignItems: 'center',
+      // ? TBD
+      [theme.breakpoints.down('xs')]: {
+        flexDirection: 'column',
+      },
+    },
+    toLabelDelimiter: {
+      margin: '0 16px',
+      [theme.breakpoints.down('xs')]: {
```

Comment:
We use `theme.breakpoints.up()` as much as possible in the codebase, up('sm') in our case.